tools/compiletest/src/runtest.rs

```diff
@@ -353,11 +354,11 @@ impl<'test> TestCx<'test> {
     /// Runs Kani on the test file specified by `self.testpaths.file`. An error
     /// message is printed to stdout if verification output does not contain
     /// the expected output in `expected` file.
-    fn run_expected_test(&self) {
+    fn run_expected_test(&self, new_format: bool) {
```

Comment:
Since this suite is temporary, I would recommend against modifying existing functions and creating a run_new_format_test(&self) with a deprecated tag already. It makes cleaning up later on much easier and less likely that people will rely on this logic.